.github/workflows/pull-request.yml

```
@@ -0,0 +1,50 @@
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Pull Request
+on:
+  push:
+    branches: ["**"]
+  pull_request:
+    branches: ["**"]
+  workflow_dispatch:
+
+env:
+  GOPATH: /opt/go
+  PATH: /opt/go/bin:/bin:/usr/bin:/sbin:/usr/sbin:/usr/local/bin:/usr/local/sbin
+  GO_VER: 1.18.2
+
+jobs:
+  verify-build:
+    name: Verify Build
+    runs-on: ubuntu-20.04
```

It might be a good time to move up to the latest Ubuntu LTS release, now that it has been the active release for a while.
```suggestion
    runs-on: ubuntu-22.04
```